test.js

```diff
@@ -0,0 +1,15 @@
+var request = require('request');
+
+request.post({
+    url: 'http://localhost:8000/metadata',
```

Comment:
Request supports `url` and `uri` interchangeably.  Since there's a `url` module, I have a tendency to use `uri` as my variable names (and as keys so it matches).